pkg/operator/ceph/cluster/mgr/spec.go

```diff
@@ -321,7 +321,14 @@ func (c *Cluster) makeDashboardService(name, activeDaemon string) (*v1.Service,
 func (c *Cluster) getPodLabels(daemonName string, includeNewLabels bool) map[string]string {
 	labels := controller.CephDaemonAppLabels(AppName, c.clusterInfo.Namespace, "mgr", daemonName, includeNewLabels)
 	// leave "instance" key for legacy usage
-	labels["instance"] = daemonName
+	if includeNewLabels {
+		labels["app.kubernetes.io/name"] = AppName
+		labels["app.kubernetes.io/instance"] = c.clusterInfo.Namespace + "_mgr-" + daemonName
```

Comment:
I would vote to keep it simple and not include the namespace in the label. The namespace is already implicit.
```suggestion
		labels["app.kubernetes.io/instance"] = "_mgr-" + daemonName
```